### DISCUSSION TITLE
Is locally importing settings json possible ?

### DISCUSSION THREAD

**Discussion by kntjspr** [kkapsner/CanvasBlocker#596] [Q&A] [answered]
I'm wondering if it's possible to load settings locally without going to the extension and click the button?

I wanted to create a program (.net specifically)  which import/export the extension settings locally without going some sort of macros to click those load buttons. 

I've tried exporting and importing the .sqlite file on profile\storage\default\{extension id} but it messes things up and those weird ^usercontextid on the folder name.

**Comment by kkapsner** [comment, marked as answer]
Sorry for the late answer.

As the settings are completely managed by Firefox at the back end I think these .sqlite files are your best guess. I did not tinker with them (they also changed with the different Firefox versions - they once were plaintext JSON files).

But I would be surprised if there weren't any tools to set up a Firefox profile with preset WebExtensions and settings for them.
